src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -69,6 +69,7 @@ private static bool IsEscapable(SymbolDisplayPartKind kind)
             {
                 case SymbolDisplayPartKind.AliasName:
                 case SymbolDisplayPartKind.ClassName:
```

Comment:
>SymbolDisplayPartKind.ClassName [](start = 21, length = 31)

Please confirm that all other places where we check for ```SymbolDisplayPartKind.ClassName``` are properly adjusted to handle new kind

Reply:
Yes, I went through the usages and adjusted them (hopefully) properly.